pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -1567,366 +670,53 @@ func (vs *VSphere) cleanUpDummyVMs(dummyVMPrefix string) {
 	// Create context
```

Comment:
Why this cleanupDummyVms is here?
Shouldn't it be handled by diskManagers.vmdm.go?